.github/workflows/main.yml

```diff
@@ -17,53 +17,76 @@ jobs:
   decision:
     name: Decision
     runs-on: ubuntu-20.04
+    outputs:
+      skipped: ${{ steps.skipDecision.outputs.result }}
+      platforms: ${{ steps.platformDecision.outputs.result }}
     steps:
-      # If an identical build exists that suceeded, skip this workflow run. We don't do
-      # this check for pull_request and merge_group events, out of caution.
-      - name: Skip previous identical builds
-        if: ${{ github.event_name != 'pull_request' && github.event_name != 'merge_group' }}
+      - name: Skip Decision
+        id: skipDecision
         uses: actions/github-script@v6
         with:
+          result-encoding: string
           script: |
+            // Never skip workflow runs for pull requests or merge groups, which might
+            // need to actually run / retry WPT tests.
+            if (context.eventName == "pull_request" || github.event_name == "merge_group") {
```

Comment:
Can `context.eventName` be replaced with `github.event_name`  as well (similar to merge_group clause)?

Reply:
No, I don't think we can. The name is a bit different for scripts inside `github-script` actions.

Reply:
Hmm....isn't the other part using `github.event_name`  of this OR clause also inside github-script action? Or do you mean value of 'pull_request' is not available outside of github-script?

Reply:
Oh, you're totally right. This is a typo that should definitely be fixed. :man_facepalming: